src/qibolab/instruments/qblox/cluster_qrm_rf.py

```diff
@@ -276,22 +260,9 @@ def _get_next_sequencer(self, port: str, frequency: int, qubits: dict, qubit: No
 
         # select a new sequencer and configure it as required
         next_sequencer_number = self._free_sequencers_numbers.pop(0)
-        if next_sequencer_number != self.DEFAULT_SEQUENCERS[port]:
-            for parameter in self.device.sequencers[
-                self.DEFAULT_SEQUENCERS[port]
-            ].parameters:
-                # exclude read-only parameter `sequence` and others that have wrong default values (qblox bug)
-                if not parameter in [
-                    "sequence",
-                    "thresholded_acq_marker_address",
-                    "thresholded_acq_trigger_address",
-                ]:
-                    value = self.device.sequencers[self.DEFAULT_SEQUENCERS[port]].get(
-                        param_name=parameter
-                    )
-                    if value:
-                        target = self.device.sequencers[next_sequencer_number]
-                        target.set(parameter, value)
+        default_sequencer_number = self.DEFAULT_SEQUENCERS[port]
+        if next_sequencer_number != default_sequencer_number:
+            self.clone_sequencer_params(default_sequencer_number, next_sequencer_number)
```

Comment:
Also, this whole pattern is repeated. In principle, you could split even this.

Reply:
On this I prefer to work when I will able to test on hardware. Also I'm really tempted to rewrite all the `process_pulse_sequence()` so idk if it's worth making it look better.